ietf-layer0-types.yang

```diff
@@ -1761,17 +1746,27 @@ module ietf-layer0-types {
         on the receiver";
     }
     list cd-penalty {
+      key cd-index;
       config false;
       description
         "Optional penalty associated with a given accumulated
         chromatic dispersion (CD) value.
 
         This list of pair cd and penalty values can be used to
         sample the function penalty = f(CD).";
+      leaf cd-index {
+        type decimal-2 {
+          range "0..max";
```

Comment:
I just have a doubt: I have no example of cases where accumulated CD would be negative, 
 so OK for me with this restriction. Are you sure that this will never happen ?